tests/assembler.rs

```diff
@@ -10,7 +10,7 @@
 extern crate solana_sbpf;
 extern crate test_utils;
 
-use solana_sbpf::program::{FunctionRegistry, SBPFVersion};
+use solana_sbpf::program::SBPFVersion;
 use solana_sbpf::vm::Config;
 use solana_sbpf::{assembler::assemble, ebpf, program::BuiltinProgram};
```

Comment:
Not for this PR, but we could configure `cargo fmt` to group imports as we do in the Agave repo.